src/lib/index.ts

```diff
@@ -67,9 +92,9 @@ class TPAStylePlugin {
     });
   }
 
-  private replaceRuntimeModule(compiler) {
+  private replaceRuntimeModule(compiler: webpack.Compiler) {
     const runtimePath = path.resolve(__dirname, '../../runtime.js');
-    const nmrp = new webpack.NormalModuleReplacementPlugin(/runtime\.js$/, resource => {
+    const nmrp = new webpack.NormalModuleReplacementPlugin(/runtime\.js$/, (resource: any) => {
```

Comment:
by default `NormalModuleReplacementPlugin` has such typings:
```
class NormalModuleReplacementPlugin extends Plugin {
    constructor(resourceRegExp: any, newResource: any);
}
```
Also, it's not vital to have type for resource here.